### DISCUSSION TITLE
Support annotation role permission to only access Annotation Queue feature

### DISCUSSION THREAD

**Discussion by ytqh** [langfuse/langfuse#3772] [Ideas]
### Describe the feature or potential improvement

Annotation Queues are great. Can you add a new role permission, like an Annotation role, to only score the traces in Annotation Queues? We want to assign the queue annotation job to some collaborators.

**Comment by marcklingen** [comment]
great idea, thanks for sharing!

**Comment by robertkirk13** [comment]
We need this. We don't want the people doing the annotation to have access to all the prompts as it's a security risk. This feature would make us switch to langfuse.

**Comment by AkioNuernberger** [comment]
@marcklingen I've gotten user feedback that they would like to set up RBAC to allow access for external reviewers to only specific annotation queues instead of being able accessing all queues in a project.

**Comment by PalvikaTR** [comment]
@marcklingen  we also need this feature and while I was trying to assign users to the annotation queue, I observed that the users who are not assigned to the queue and have in general access to the project can also go ahead and process the queue. My understanding was that only whoever is assigned to a particular queue would be able to annotate and process the queue. Please correct me if my understanding is wrong.

**Comment by camorales197** [comment]
Any update on this? It would be a game-changer in terms of compliance.

**Comment by pauloeavf** [comment]
This feature would definitely increase enterprise adoption. It's a hard sell to expose all the workflow details with SMEs that need to annotate traces.
